VaticanBAV/borget/BAVborget9.xml

```diff
@@ -0,0 +1,190 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BAVborget9" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malkǝʾ Hymns and Gadla ʾƎwosṭātewos</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AD"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                  <msIdentifier>
+                    <repository ref="INS0003BAV"/>
+                    <collection>Borgiani aethiopici</collection>
+                    <idno facs="https://digi.vatlib.it/iiif/MSS_Borg.et.9/manifest.json">Vat. Borg. et. 9</idno>
+                  </msIdentifier>
+
+                  <msContents>
+                    <summary/>
+
+                    <msItem xml:id="ms_i1">
+                      <locus from="5r" to="34r"/>
+                      <title ref="LIT2393tabiba"/>
+                      <textLang mainLang="gez"/>
+                      <note>Divided according to the seven days of the week.</note>
+                    </msItem>
+
+                    <msItem xml:id="ms_i2">
+                      <locus from="34v" to="44v"/>
+                      <title ref="LIT2842RepCh125"/>
+                      <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i3">
+                      <locus from="44v" to="53r"/>
+                      <title ref="LIT2444Temher"/>
+                      <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i4">
+                      <locus from="53v" to="64v"/>
+                      <title ref="LIT1438GadlaE">Ten Miracles of St ʾEwosṭātewos with hymns.</title>
```

Comment:
not LIT4315Miracles?

Reply:
Check also LIT3200DossierEw as the ID for combination of various hagiographic works dedicated to Ewostatewos (Gadl, Miracles, Hymns, in any constellation)

Reply:
It is a different set of miracles than those included in LIT4315Miracles. According to a comment in LIT4315Miracles, Turaiev's edition included 16, which also seems to be a different recension.

The catalogue here notes a similarity with the fragment BL Or 829*, but it is only the incipit, it otherwise seems to be a different text.

I haven't been able to get access to Lusini's article on the manuscript tradition, and I'm not sure that there are work records for all the various recensions. This, in any case, seems different than is typical.

Reply:
We have at least three records for the various recensions 4319 alpha 4320 beta 4321 gamma
Lusini's article I think has been superseded by his PhD monograph

Reply:
As to the number of miracles - this varies, I had MSS with 11 in EthioSPaRe etc, LIT4315Miracles is a general label, we have not gotten to identify single miracles

Reply:
Looking more closely at the text in the images, it appears that it is a versified version of the Miracles, which explains why they are all so short.

Reply:
It would seem best not to follow Grébaut's division of the text here and instead to use the divisions of Chaîne, listing these as 1.4.1 (መንክራቲከ፡ ሰማዕነ፡), 1.4.2 (RepCh 323), 1.4.3 (RepCh 333), and 1.4.4 (RepCh 353).

Reply:
Sounds good. Can you check the refs in BAVet46? Don't look right to me.
For completeness I have pushed to the same branch an E-S ms with the miracles (facs added, to see the images)

Reply:
An amazing coincidence, @Ralph-Lee-UK just sent me images for EMIP 764 and it also contains (84r-86r) this same additional stanza at the beginning ("መንክራቲከ፡ ሰማዕነ፡ እንዘ፡ ትነብር፡ አርማንያ፡ መካነ።") followed by Chaîne 323, 333, and 353 and then some additional stanzas.

Reply:
Since this seems to be a frequent constellation we might mention it in the work record for miracles, or consider creating a new overarching work record for the collection miracles+hymns

Reply:
I was just thinking about this, a work record for "Versified Miracles of ʾEwosṭātewos" or the like, with textparts for each of the constituent parts.

BL Or 641 titles it "ኡደተ፡ ባሕር፡", but the MS only contains RepCh 353.

Reply:
Or maybe better, መንክራቲከ፡ + 323 + 333 and then separately 353 (ዑደተ፡ ባሕር፡). In most manuscripts, while these are all copied together, there is a clear separation between 353 and what precedes it (no clear separation with the other parts).